xarray/core/alignment.py

```diff
@@ -35,16 +35,17 @@ def _get_joiner(join):
 
 
 def align(*objects, **kwargs):
-    """align(*objects, join='inner', copy=True, indexes=None,
-             exclude=frozenset())
+    """align(*objects, join='inner', copy=True, fill_value=dtypes.NA,
+             indexes=None, exclude=frozenset())
```

Comment:
One small additional clean-up you could do here: since we no longer support Python 2, these function signature could be moved from the docstring onto the function itself, e.g.,
```
def align(*objects, join='inner', copy=True, fill_value=dtypes.NA,
             indexes=None, exclude=frozenset())"
```

This would let you drop the calls to `kwargs.pop()` below, and would be a small win for efficiency/clean code.